coc/events.pyi

```diff
@@ -137,6 +139,34 @@ class WarEvents:
     @classmethod
     def new_war(cls, tags: Iterable = None, custom_class: Type[ClanWar] = ClanWar, retry_interval: int = None) -> _EventDecoratorReturn: ...
 
+class RaidEvents:
+    event_type: str
+    @classmethod
+    def new_member(cls, tags: Iterable = None, custom_class: Type[RaidLogEntry] = RaidLogEntry, retry_interval: int = None) -> _EventDecoratorReturn: ...
+    @classmethod
+    def offensive_opponent(cls, tags: Iterable = None, custom_class: Type[RaidLogEntry] = RaidLogEntry, retry_interval: int = None) -> _EventDecoratorReturn: ...
+    @classmethod
+    def defensive_opponent(cls, tags: Iterable = None, custom_class: Type[RaidLogEntry] = RaidLogEntry, retry_interval: int = None) -> _EventDecoratorReturn: ...
+    @classmethod
+    def attack_district_destruction_change(cls, tags: Iterable = None, custom_class: Type[RaidLogEntry] = RaidLogEntry, retry_interval: int = None) -> _EventDecoratorReturn: ...
+    @classmethod
+    def defense_district_destruction_change(cls, tags: Iterable = None, custom_class: Type[RaidLogEntry] = RaidLogEntry, retry_interval: int = None) -> _EventDecoratorReturn: ...
```

Comment:
I would rename these two to `[offensive|defensive]_district_destruction` to be more in line with the other namings.